jobs/sql/extractions/patient_last_update_date.sql

```diff
@@ -0,0 +1,2 @@
+SET @partition = '${partitionNum}';
+SELECT CONCAT(@partition, '-', patient_id) as patient_id, last_updated, deleted from dbevent_patient;
```

Comment:
This loads a table of patients that have changed based on db_event (captured by the pihcore module).  This is added once for the entire pipeline early on, as it is a dependency of the incremental jobs.  Any incremental job can then use it, it doesn't need to be done more than once.